case-server/src/main/java/com/powsybl/caseserver/EntsoeCaseInfos.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.caseserver;
+
+import com.powsybl.entsoe.util.EntsoeGeographicalCode;
+import com.powsybl.iidm.network.Country;
+import io.swagger.annotations.ApiModel;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.experimental.SuperBuilder;
+import org.joda.time.DateTime;
+import org.springframework.messaging.Message;
+
+/**
+ * A utility class to work with Entsoe file name parsers.
```

Comment:
It is more a data class than a utility class?

Reply:
Changed by this :
A class to store metatada for Entsoe file name parser